tox.ini

```diff
@@ -34,54 +38,54 @@ whitelist_externals =
 description = Runs all non-Python3-based server unit/functional tests
 passenv = PBENCH_UNITTEST_SERVER_MODE
 deps =
-   -r{toxinidir}/server/requirements.txt
-   -r{toxinidir}/server/test-requirements.txt
+    -r{toxinidir}/server/requirements.txt
+    -r{toxinidir}/server/test-requirements.txt
 commands =
     bash -c "./server/bin/unittests {posargs}"
 
 [testenv:datalog]
 description = Runs all non-Python3-based agent tool-scripts/datalog unit/functional tests
 deps =
-   -r{toxinidir}/agent/requirements.txt
-   -r{toxinidir}/agent/test-requirements.txt
+    -r{toxinidir}/agent/requirements.txt
+    -r{toxinidir}/agent/test-requirements.txt
 commands =
     bash -c "./agent/tool-scripts/datalog/unittests {posargs}"
 
 [testenv:postprocess]
 description = Runs all non-Python3-based agent tool-scripts/postprocess unit/functional tests
 deps =
-   -r{toxinidir}/agent/test-requirements.txt
+    -r{toxinidir}/agent/test-requirements.txt
```

Comment:
ditto

Reply:
It makes all the indentation in the file consistent.